src/main/resources/application.yaml

```diff
@@ -70,8 +66,16 @@ rp:
   jwt:
     signing-key: 123
 
+  datasource:
+    type: com.zaxxer.hikari.HikariDataSource
+    driverClassName: org.postgresql.Driver
+    jdbcUrl: ${rp.db.url}
+    username: ${rp.db.user}
+    password: ${rp.db.pass}
+    maximumPoolSize: 27
+
   db:
-    url: jdbc:postgresql://${rp.db.host}:${rp.db.port}/${rp.db.name}?user=${rp.db.user}&password=${rp.db.pass}
+    url: jdbc:postgresql://${rp.db.host}:${rp.db.port}/${rp.db.name}
```

Comment:
why did you remove user and password from here?

Reply:
This statement was split into 3 key-value statements:
url: url
username: username
password: password